bam_readcount_helper.py

```diff
@@ -24,9 +24,9 @@ def filter_sites_in_hash(region_list, bam_file, ref_fasta, output_dir, insertion
     else:
         output_file = os.path.join(output_dir, 'bam_readcount_snv.tsv')
     bam_readcount_cmd.append(bam_file)
-    execution = run(bam_readcount_cmd, stdout=PIPE, check=True)
+    execution = Popen(bam_readcount_cmd, stdout=PIPE)
     with open(output_file, 'wb') as output_fh:
-        output_fh.write(execution.stdout)
+        output_fh.write(execution.communicate()[0])
```

Comment:
Do we need to check `execution.returncode` ourselves after this line to see if anything went wrong (like the `check=True` in the python 3 version)?